.gitignore

```diff
@@ -1,4 +0,0 @@
-*.swp
```

Comment:
The `.swp` ignore was  there before I added the other stuff.

Reply:
Yes, don't worry :wink: 
I was just a bit frustrated that this .gitignore was just getting filled with various tmp files coming from stuff which have basically nothing to do with ssowat itself and i don't think we want to have 10+ lines filled with that kind of stuff. Anyway that's not a big deal but I propose to just get rid of the .gitignore

Reply:
Yes, I can also put all the stuff into my global ignore file. (Which would be best practice anyway :p)